src/main/java/seedu/address/logic/commands/AddTagPersonCommand.java

```diff
@@ -0,0 +1,151 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Adds tags to an existing person in the address book.
+ */
+public class AddTagPersonCommand extends Command {
+
+    public static final String COMMAND_WORD = "addTagPerson";
+    public static final String SHORTENED_COMMAND_WORD = "atagp";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " (alias: " + SHORTENED_COMMAND_WORD + ")"
+            + ": Adds new tag to the person identified "
+            + "by the index number used in the displayed contact list. \n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_TAG + "catering";
```

Comment:
```suggestion
            + PREFIX_TAG + "catering" + " " + PREFIX_TAG + "dining";
```
Similar comment in AddTagTaskCommand